contrib/runners/orquesta_runner/orquesta_runner/runner.yaml

```diff
@@ -4,6 +4,11 @@
   enabled: true
   name: orquesta
   runner_module: orquesta_runner
+  runner_parameters:
+    notify:
+      type: array
```

Comment:
Probably also not a bad idea to specify type for each array item aka ``items: type: string``.